ReactQt/runtime/src/rootview.cpp

```diff
@@ -232,8 +234,14 @@ void ReactView::bridgeReady() {
     }
 }
 
-void ReactView::componentComplete() {
-    Q_D(ReactView);
+void RootView::requestPolish() {
+    if (ReactAttachedProperties::get(this)->tag() == -1)
```

Comment:
It might make sense to wrap this check into method with easy to ready and understand title.